app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.tsx

```diff
@@ -118,6 +119,8 @@ export const AppLayoutWithSidebar = ({
                   {page.name}
                 </LinkNav>
               ))}
+
+              <SidebarActions experiment={experiment} />
```

Comment:
Might want to update some of the AppLayoutWithSidebar stories to include `can_archive`, because [the action appears disabled in all cases right now](https://github.com/mozilla/experimenter/blob/105e4d86f33b8159732c2c130b3b51b6a81338ef/app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.stories.tsx).